### PR TITLE
Add new apps to the catalog

### PR DESCRIPTION
**Pull request by dherault**
👍 

Added or modified: 
```js
  'airbyte',
  'console',
  'airflow',
  'argo-cd',
  'argo-workflows',
  'chatwoot',
  'crossplane',
  'dagster',
  'datahub',
  'etcd',
  'filecoin',
  'ghost',
  'gitlab',
  'grafana',
  'grafana-tempo',
  'growthbook',
  'hasura',
  'influx',
  'ingress-nginx',
  'istio',
  'jitsu',
  'kafka',
  'knative',
  'kserve',
  'kubecost',
  'kubeflow',
  'kubescape',
  'metabase',
  'minecraft',
  'minio',
  'mlflow',
  'mongodb',
  'mysql',
  'n8n',
  'nextcloud',
  'nocodb',
  'oncall',
  'plural',
  'postgres',
  'rabbitmq',
  'ray',
  'redis',
  'reloader',
  'rook',
  'sentry',
  'spark',
  'superset',
  'terraria',
  'touca',
  'trino',
  'valheim',
  'vault',
  'vaultwarden',
```

Missing:
- oncall
- grafana-tempo

For script failure reasons.

### CODE REVIEW

**Comment by dherault** [comment]
Ohhhh, everything broke, need to speak with Michael about the script :/

**Comment by dherault** [comment]
I fixed it 💪

**Comment by dherault** [comment]
mmm @dogmar how do I add the new pages to the sidebar hierarchy?

File: pages/repositories/airbyte.md [hunk 1]
```diff
@@ -1,59 +1,43 @@
...
 
-We currently support this repository on the following infrastructure providers:
+We currently support Airbyte for the following providers:
 
 {% tabs %}
 
 {% tab title="AWS" %}
-```shell
 plural bundle install airbyte airbyte-aws
-```
```
Comment (dogmar) [inline pages/repositories/airbyte.md:20]: @dherault These commands should be surrouned by a code fence.
````
```shell
{COMMAND}
```

**Review by dogmar — CHANGES_REQUESTED** [review]

**Comment by dherault** [comment]
Closing in favor of @dogmar 's work

**Pull request closed without merging**